lib/tasks/manage_admins.rake

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+desc "Manage admin users"
+namespace :admin do
+  desc "Create admin user"
+  task :create, %i[full_name email] => :environment do |_task, args|
+    include Rails.application.routes.url_helpers
+    default_url_options[:host] = Rails.configuration.domain
+
+    puts "Creating user"
```

Comment:
I would much prefer it if all this task did was pass the params to another object that does the actual creating along the same lines as the data imports. One option could be to define a CreateAdmin class that does what we are doing in here. That would be more reusable in the controllers too. I didn't do it when I worked on the controller coz the code was just being used once.

Reply:
What about a static method on AdminProfile?

Reply:
That would work too. Might be better actually